packages/docmanager-extension/src/index.ts

```diff
@@ -711,69 +788,38 @@ function addCommands(
     }
   });
 
-  app.contextMenu.addItem({
-    command: CommandIDs.rename,
-    selector: '[data-type="document-title"]',
-    rank: 1
-  });
-  app.contextMenu.addItem({
-    command: CommandIDs.clone,
-    selector: '[data-type="document-title"]',
-    rank: 2
-  });
-  app.contextMenu.addItem({
-    command: CommandIDs.showInFileBrowser,
-    selector: '[data-type="document-title"]',
-    rank: 3
-  });
-  app.contextMenu.addItem({
-    command: CommandIDs.closeOtherTabs,
-    selector: '[data-type="document-title"]',
-    rank: 4
-  });
   app.contextMenu.addItem({
     command: CommandIDs.closeRightTabs,
     selector: '[data-type="document-title"]',
     rank: 5
   });
-  // .jp-mod-current added so that the console-creation command is only shown on the current document.
-  // Otherwise it will delegate to the wrong widget.
-  app.contextMenu.addItem({
-    command: 'filemenu:create-console',
-    selector: '[data-type="document-title"].jp-mod-current',
-    rank: 6
-  });
 
-  [
-    CommandIDs.openDirect,
-    CommandIDs.save,
-    CommandIDs.reload,
-    CommandIDs.restoreCheckpoint,
-    CommandIDs.saveAs,
-    CommandIDs.clone,
-    CommandIDs.close,
-    CommandIDs.closeAllFiles,
-    CommandIDs.closeOtherTabs,
-    CommandIDs.closeRightTabs,
-    CommandIDs.toggleAutosave
-  ].forEach(command => {
-    palette.addItem({ command, category });
-  });
+  if (palette) {
+    [
+      CommandIDs.close,
+      CommandIDs.closeAllFiles,
+      CommandIDs.closeOtherTabs,
+      CommandIDs.closeRightTabs,
+      CommandIDs.toggleAutosave
```

Comment:
`toggleAutosave` and `close` are duplicated in the above palette additions and here.

Reply:
Oh good catch. I can't fix that right this second. If you feel comfortable doing so, please do. If not, I'll be able to later this evening.

Reply:
Sure, I can fix it.

Reply:
Cheers!